install.sh

```diff
@@ -61,6 +67,9 @@ tty_bold="$(tty_mkbold 39)"
 tty_reset="$(tty_escape 0)"
 
 have_sudo_access() {
+  if [[ ! -z "${NONINTERACTIVE-}" ]]; then
```

Comment:
maybe `! -z` -> `-n`?

Reply:
Yes, shellcheck already caught me on this!